devsetup/edpm/services/dataplane_v1beta1_openstackdataplaneservice_reposetup.yaml

```diff
@@ -4,22 +4,21 @@ kind: OpenStackDataPlaneService
 metadata:
   labels:
     app.kubernetes.io/name: openstackdataplaneservice
-    app.kubernetes.io/instance: openstackdataplaneservice-sample
+    app.kubernetes.io/instance: openstackdataplaneservice-repo-setup
     app.kubernetes.io/part-of: dataplane-operator
     app.kubernetes.io/managed-by: kustomize
-    app.kubernetes.io/created-by: dataplane-operator
+    app.kubernetes.io/created-by: install_yamls
   name: repo-setup
 spec:
   label: dataplane-deployment-repo-setup
   play: |
-    - hosts: all
-      tasks:
-        - name: Enable podified-repos
-          shell: |
-            rpm -q git || sudo yum -y install git
-            sudo yum -y install python-setuptools python-requests python3-pip
-            git clone https://github.com/openstack-k8s-operators/repo-setup
-            pushd repo-setup
-            sudo python3 setup.py install
-            popd
-            sudo /usr/local/bin/repo-setup current-podified-dev
+    - name: Enable podified-repos
```

Comment:
it seems that it's not a play anymore since "hosts" and "tasks" definition are missing. Is it correct?

Reply:
I copied the style of [edpm-play](https://github.com/openstack-k8s-operators/install_yamls/blob/main/devsetup/edpm/edpm-play.yaml#L10-L11) because the dataplane-operator adds the inventory to the AEE job

Reply:
yeah, you were right, I added it back